README.md

```diff
@@ -9,7 +9,9 @@ open-source software and Google Cloud Platform.
 
 ## Google Cloud Storage connector for Apache Hadoop (HCFS)
 
-[![Maven Central](https://img.shields.io/maven-central/v/com.google.cloud.bigdataoss/gcs-connector/hadoop3.svg?label=Maven%20Central)](https://search.maven.org/search?q=g:com.google.cloud.bigdataoss%20AND%20a:gcs-connector%20AND%20v:3.1.0)
+[![Maven Central](https://img.shields.io/maven-central/v/com.google.cloud.bigdataoss/gcs-connector/hadoop1.svg?label=Maven%20Central)](https://search.maven.org/search?q=g:com.google.cloud.bigdataoss%20AND%20a:gcs-connector%20AND%20v:hadoop1-*)
```

Comment:
latest version of connector i.e. gcs-connector-3.0 , 3.1 will not be listed in this

Reply:
Need a change in the readme template. This commit is auto created and hence do to want to make changes. Will create a separate PR to fix this.